include/effects_internal.h

```diff
@@ -3,7 +3,7 @@
 
 #include "effects.h"
 
-s32 func_E0200000(s32);
+s32 gfx_rand_int(s32);
```

Comment:
Not quite sure I'd call this "gfx" as that's often what we use to refer to things that are directly interfacing with the n64 graphics hardware. Its a silly differentiation, but these graphical effects are just referred to as "fx" or "effect" I think "effect_rand_int" is probably a good name.